benchmark/README.md

```diff
@@ -31,8 +33,145 @@ npm run benchmark -- -u 4 -e 15 -r 5
 - `npm run benchmark` to run the benchmark with the default setting, to see the available flags run
 - `npm run benchmark -- --help`
 
-## Comming soon
+## Custom Tasks
+
+DiscoJS currently provides several pre-define popular tasks such as titanic, simple-face and cifar10. In order
+to understand how to add your own custom task, we will go over how we added simple-face to discojs.
+
+### Task
```

Comment:
this whole subsection here is very nice and useful, but it seems to be not node specific. we can put this in a more general readme how to define tasks (which can later be used both in browser and node)